src/puppetlabs/ezbake/core.clj

```diff
@@ -1,19 +1,21 @@
 (ns puppetlabs.ezbake.core
-  (:import (java.io File)
-           (java.util.jar JarEntry))
+  (:import (java.io File InputStream InputStreamReader)
+           (java.util.jar JarEntry JarFile))
   (:require [me.raynes.fs :as fs]
             [clojure.java.io :as io]
             [clojure.string :as str]
             [clj-time.local :as local-time]
             [stencil.core :as stencil]
             [leiningen.core.main :as lein-main]
             [leiningen.deploy :as deploy]
+            [leiningen.core.classpath :as lein-classpath]
             [leiningen.uberjar :as uberjar]
             [schema.core :as schema]
             [schema.utils :as schema-utils]
             [puppetlabs.ezbake.dependency-utils :as deputils]
             [puppetlabs.ezbake.exec :as exec]
-            [puppetlabs.config.typesafe :as ts]))
+            [puppetlabs.config.typesafe :as ts]
+            [leiningen.core.project :as project]))
```

Comment:
looks like the list is pretty sorted, so this should go with the other lein requires